symfony/framework-bundle/3.3/public/index.php

```diff
@@ -21,7 +21,9 @@
     Debug::enable();
 }
 
-// Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
+if (($trustedProxies = getenv('TRUSTED_PROXIES')) && ($trustedHeaders = getenv('TRUSTED_HEADERS'))) {
```

Comment:
The second parenthesis around `$trustedHeaders = getenv('TRUSTED_HEADERS')` are useless.